resources/i18n/ko_KR/cura.po

```diff
@@ -5,66 +5,48 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Cura 3.3\n"
+"Project-Id-Version: Cura 3.2\n"
 "Report-Msgid-Bugs-To: r.dulek@ultimaker.com\n"
-"POT-Creation-Date: 2018-03-29 08:36+0200\n"
-"PO-Revision-Date: 2018-04-11 14:40+0100\n"
-"Last-Translator: Bothof <info@bothof.nl>\n"
+"POT-Creation-Date: 2018-01-29 09:48+0000\n"
+"PO-Revision-Date: 2018-04-12 18:11+0900\n"
+"Last-Translator: Jinbuhm Kim <jbkim@ateamventures.com>\n"
 "Language-Team: Korean\n"
 "Language: ko_KR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=1; plural=0;\n"
-"X-Generator: Poedit 2.0.4\n"
+"X-Generator: Poedit 2.0.6\n"
 
-#: /home/ruben/Projects/Cura/plugins/MachineSettingsAction/MachineSettingsAction.py:22
+#: /home/ruben/Projects/Cura/plugins/MachineSettingsAction/MachineSettingsAction.py:26
 msgctxt "@action"
 msgid "Machine Settings"
-msgstr "기계 설정"
+msgstr "기기 설정"
 
 #: /home/ruben/Projects/Cura/plugins/XRayView/__init__.py:12
 msgctxt "@item:inlistbox"
 msgid "X-Ray view"
-msgstr "X 선보기"
+msgstr "엑스레이 뷰"
 
 #: /home/ruben/Projects/Cura/plugins/X3DReader/__init__.py:13
 msgctxt "@item:inlistbox"
 msgid "X3D File"
 msgstr "X3D 파일"
 
 #: /home/ruben/Projects/Cura/plugins/GCodeWriter/__init__.py:16
-#: /home/ruben/Projects/Cura/plugins/GCodeProfileReader/__init__.py:14
-#: /home/ruben/Projects/Cura/plugins/GCodeReader/__init__.py:14
 msgctxt "@item:inlistbox"
-msgid "G-code File"
-msgstr "G-code 파일"
-
-#: /home/ruben/Projects/Cura/plugins/ModelChecker/ModelChecker.py:30
-msgctxt "@info:title"
-msgid "Model Checker Warning"
-msgstr "모델 검사기 경고"
-
-#: /home/ruben/Projects/Cura/plugins/ModelChecker/ModelChecker.py:66
-#, python-brace-format
-msgctxt "@info:status"
-msgid ""
-"Some models may not be printed optimally due to object size and chosen material for models: {model_names}.\n"
-"Tips that may be useful to improve the print quality:\n"
-"1) Use rounded corners.\n"
-"2) Turn the fan off (only if there are no tiny details on the model).\n"
-"3) Use a different material."
-msgstr "일부 모델은 개체 크기 및 다음 모델에 대해 선택한 재료로 인해 최적으로 인쇄되지 않을 수 있습니다. {model_names}.\n인쇄 품질을 향상하는 데 유용한 팁:\n1) 둥근 모서리를 사용합니다.\n2) 팬을 끕니다(모델에 대해 세부 정보가 거의 없는 경우에만).\n3) 다른 재료를 사용합니다."
```

Comment:
:( It looks like you've translated Cura 3.2 instead of Cura 3.3. These translations are outdated!

Reply:
Hmm.  Ok, I'll check it again.